gordo/server/views/base.py

```diff
@@ -243,10 +243,19 @@ class ModelListView(Resource):
     List the current models capable of being served by the model
     """
 
-    @api.doc(description="List the metadata for all models capable of being served.")
+    @api.doc(
+        description="List the names for all models capable of being "
+        "served, and those expected to be able to serve."
+    )
     def get(self, gordo_project: str):
         available_models = os.listdir(g.collection_dir)
-        return jsonify({"models": available_models})
+        return jsonify(
+            {
+                "models": available_models,
+                "server-version": __version__,
+                "expected-models": current_app.config["EXPECTED_MODELS"],
```

Comment:
so yeah, this was a bit tricky... `current_app.config["EXPECTED_MODELS"]` is not the expected models for a old revision, only for the current. So what do return for old revision? Remove the key? Should it just be the same as models? Should it be empty? Uffda, IDK. 
Also, I noobed when I in my previous comment said "current version", I meant current revision. So one can

Reply:
Could maybe change the key to something like `current-revision-expected-models`?